media-gfx/ahoviewer/ahoviewer-1.6.2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_EAUTORECONF="yes"
+
+inherit gnome2 xdg-utils
+
+DESCRIPTION="A GTK2 image viewer, manga reader, and booru browser"
+HOMEPAGE="https://github.com/ahodesuka/ahoviewer"
+SRC_URI="https://github.com/ahodesuka/ahoviewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl_ssl_gnutls +curl_ssl_openssl +gstreamer libsecret +rar +ssl +zip"
```

Comment:
Please don't multiply those horrible CURL_SSL flags. They're PITA to anyone using Gentoo.

Reply:
So, switch back to having `gnutls` and `ssl` USE flags, then make 
`REQUIRED_USE="?? ( gnutls ssl )"`
? It's a lot simpler, too. Upstream does say "curl must be built with the same backend", 
https://github.com/ahodesuka/ahoviewer/blob/master/configure.ac#L44
https://github.com/ahodesuka/ahoviewer/blob/master/configure.ac#L52

so with those USE flags, I could depend on curl with correct backend, as I originally did.
```
gnutls? ( net-misc/curl[curl_ssl_gnutls] )
ssl? ( net-misc/curl[curl_ssl_openssl] )
```

Reply:
Yep. But you should also dep on gnutls/openssl directly, with `:=`/slot as appropriate.